portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,103 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;
+}
+
+button {
+  background: none;
+	border: none;
```

Comment:
Nit: The indentation looks off in this line.

Reply:
Fixed!